cypress-tests/cypress/e2e/admin/pageBuilder/menus/menuCrud.cy.ts

```diff
@@ -0,0 +1,59 @@
+context("Page Builder - Menu CRUD", () => {
+    const menuName = "Test menu";
+    const menuSlug = "test-slug";
+
+    const menuNameEdit = "Testing Menu123";
+    const menuDescEdit = "This is an edited description.";
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllMenus();
+    });
+
+    it("should be able to create, edit, and immediately delete a menu", () => {
+        cy.visit("/page-builder/menus");
+
+        // Create a menu.
+        cy.findByTestId("data-list-new-record-button").should("exist");
+        cy.findByTestId("data-list-new-record-button").click();
+
+        cy.findByTestId("pb.menu.create.name").type(menuName);
+        cy.findByTestId("pb.menu.save.button").click();
+        cy.findByText("Value is required.").should("exist");
+        cy.findByTestId("pb.menu.create.slug").type(menuSlug);
+        cy.findByTestId("pb.menu.create.description").type("This is a cool test.");
+        cy.findByTestId("pb.menu.save.button").click();
+
+        cy.findByText("Menu saved successfully.").should("exist");
+        cy.wait(500);
```

Comment:
Let's not use cy.wait if really not needed. If needed, let's ensure a comment explains what it's there.

Let's adopt the same approach in all future tests.